### ISSUE TITLE
Trending Developers

### ISSUE THREAD

**Issue by iShawnWang** [rhysd/node-github-trend#11]
# Awesome lib :D

May be also scraping [https://github.com/trending/developers](https://github.com/trending/developers)  ~

**Comment by rhysd** [comment]
Definitely. But I have no plan to use developers information extracted from the page. So this feature may be implemented in the future, but not urgent for me. Of course a pull request is welcome.
